mscore/pagesettings.h

```diff
@@ -22,29 +22,42 @@
 #define __PAGESETTINGS_H__
 
 #include "ui_pagesettings.h"
+#include "libmscore/mscore.h"
+#include "libmscore/page.h"
 
 namespace Ms {
 
 class MasterScore;
 class Score;
 class Navigator;
 
+// For converting from Ms::Units to a two-char string used as a suffix
+static const char* unitSuffixes[] = {"px", "mm", "in", "sp"};
```

Comment:
thinink about it some more: that are quite a few places we we use unit suffixes (and visible in the UI and some translated), in addition to the ones here we use "%", "" (i.e. an empty string) and "pt" (for font size in Point). Maybe we could once and forever collect and combine them in just one spot? Also only translate them once...